oci-seccomp-bpf-hook.go

```diff
@@ -171,7 +175,18 @@ func startFloatingProcess() error {
 		return err
 	}
 	pid := s.Pid
-	fileName := s.Annotations["io.containers.trace-syscall"]
+	var outputFile string
+	var inputFile string
+	annotation := s.Annotations["io.containers.trace-syscall"]
+	annotationSplit := strings.SplitN(annotation, ";", 2)
+	logrus.Println(annotationSplit)
+	for _, paths := range annotationSplit {
+		if strings.HasPrefix(paths, "if:") {
+			inputFile = strings.TrimSpace(strings.TrimPrefix(paths, "if:"))
+		} else if strings.HasPrefix(paths, "of:") {
```

Comment:
No need for the else here, but I guess it saves CPU.
You could also add a continue,  but I am bike shedding.